app/models/complaint.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+# Complaint data appears in its CSV as list of individual student veteran complaints keyed by an optional facility code.
```

Comment:
Can we put this comment in a Markdown within docs/ folder and then just link to it here? I think someone reading the code could figure this out without the comment actually, but if anything having this really long comment before getting to the code is not desirable.